### PR TITLE
opti: optimizing webpack compire time 

### PR DESCRIPTION
**Pull request by rainfu**
| Q                | A
| ---------------- | ---
| Bug fix?         | no
| New feature?     |no
| License          | MIT

### Description

optimizing webpack compire time from 60s to <30s

|          | current | eslint-loader opt | autodll-webpack |
| -------- | ------- | ----------------- | --------------- |
| main     | 27s,28s | 28s,18s           | 3s,3s           |
| renderer | 30s,31s | 21s,19s           | 21s,20s         |
| total    | 57s,59s | 49s,37s           | 24s,23s         |

**note**
- i have test it in windows within dev and prod env,should test on mac or other env
- next we can autodll the nodejs env modules
- next we can autodll the muya assets libs js file

### CODE REVIEW

File: .electron-vue/webpack.renderer.config.js [hunk 1]
```diff
@@ -160,12 +212,23 @@ const rendererConfig = {
...
-    new VueLoaderPlugin()
+    new VueLoaderPlugin(),
+    new AutoDllPlugin({
+      context: path.join(__dirname, '..'),
+      inject: true, // will inject the DLL bundles to index.html
+      debug: true,
+      filename: '[name].js',
+      entry: {
+        vendor: autoDllModules
+        // muyaLib: autoDllMuyaLib
```
Comment (Jocs) [inline .electron-vue/webpack.renderer.config.js]: Hi, why commented this line of code?

**Comment by Jocs** [comment]
@rainfu This PR significantly saves all contributors time, thank you very much for your contribution, but when I pull your code locally, the runtime reports an error, as follows:

```js
Uncaught ReferenceError: vendor_9894a12e780d86e7f1ef is not defined
    at eval (eval at dll-reference vendor_9894a12e780d86e7f1ef (renderer.js:10731), <anonymous>:1:18)
    at Object.dll-reference vendor_9894a12e780d86e7f1ef (renderer.js:10731)
    at __webpack_require__ (renderer.js:768)
    at fn (renderer.js:131)
    at eval (webpack-internal:///./node_modules/webpack/buildin/module.js:1)
    at Object../node_modules/webpack/buildin/module.js (renderer.js:5945)
    at __webpack_require__ (renderer.js:768)
    at fn (renderer.js:131)
    at eval (webpack-internal:///./node_modules/webpack-hot-middleware/client.js?noInfo=true&reload=true:308)
    at Object../node_modules/webpack-hot-middleware/client.js?noInfo=true&reload=true (renderer.js:5901)
```

**Comment by fxha** [comment]
Thank you for your contribution @rainfu, I'm not a JS expert but are there some reasons to use `autodll-webpack-plugin` instead of `hard-source-webpack-plugin` that is also used by webpack 5 like described in the readme?

**Comment by Jocs** [comment]
@rainfu conflicted. and the CI also failed.

**Comment by Jocs** [comment]
@rainfu 这个 PR 有冲突了，请先解决冲突。😁

**Comment by rainfu** [comment]
@Jocs @fxha 
sorry, perhaps this pull request isn't stable enough, i cancel this it and opti in future .

**Pull request closed without merging**